Jenkinsfile

```diff
@@ -458,6 +458,10 @@ pipeline {
             name: "TARGET_NAVI21",
             defaultValue: true,
             description: "")
+        booleanParam(
+            name: "TARGET_NAVI32",
+            defaultValue: true,
```

Comment:
[Q] So we have Navi32 in our CI pool? Great!

Reply:
Yes!~ limit quantity so this stage is under experiment.